web/src/stream_data.ts

```diff
@@ -456,7 +456,10 @@ export function canonicalized_name(stream_name: string): string {
     return stream_name.toString().toLowerCase();
 }
 
-export function get_color(stream_id: number): string {
+export function get_color(stream_id: number | undefined): string {
+    if (stream_id === undefined) {
+        return DEFAULT_COLOR;
+    }
```

Comment:
This change looks like it's a bit lost?

Reply:
I did this because I was doing this for the stream bar anyways, and `get_color` is called from `drafts` with an undefined stream_id. I can do the same thing as in the stream bar PR and export `DEFAULT_COLOR`, like this:

```
        const draft_stream_color = draft.stream_id === undefined? DEFAULT_COLOR : stream_data.get_color(draft.stream_id);
```

Though I still prefer the solution I have here instead of having to add `DEFAULT_COLOR` to multiple callsites. Thoughts?

Reply:
If we keep this change to `get_color` I can make it a separate commit though.

Reply:
@timabbott would you prefer that `drafts.ts` do something like this? 

```
const draft_stream_color = draft.stream_id === undefined? DEFAULT_COLOR : stream_data.get_color(draft.stream_id);
````

Or, would you prefer that `get_color` return `DEFAULT_COLOR` as I changed it here? If so, I can adjust the call to `get_color` in `stream_bar.decorate` as well, and can put that in a separate commit.

Reply:
I kept the change and put it in a separate commit for now, but happy to talk about doing something different.

Reply:
I'm not sure about this but don't think it's important.